### PR TITLE
feat(ui): Navbar Improvements

### PR DESCRIPTION
**Pull request by asharonbaltazar**
- add dnd to alert presets
- add delete button to alert presets
- updated text and background colors 
- add disclosure to hide categories

### CODE REVIEW

**Comment by asharonbaltazar** [comment]
https://github.com/keephq/keep/assets/58940073/3e0147d1-66d3-4b50-8db0-2e8049a53c32

**Comment by talboren** [comment]
Some general comments from first sight:
1. Correlation and dismissed share the same icon, we should change that.
2. Trash icon is always shown, maybe it should only be on hover?
3. When hovering over the trash icon, perhaps it should have a color as well?
4. If we removed "delete" of preset to this trash icon, remove the delete from next to the filter input

File: keep-ui/app/dark-mode-toggle.tsx [hunk 1]
```diff
@@ -43,7 +43,7 @@ export default function DarkModeToggle() {
 
   return (
     <button
-      className="flex items-center justify-between space-x-3 w-full text-sm p-1 text-gray-700 hover:bg-gray-200 font-medium rounded-lg hover:text-orange-500 focus:ring focus:ring-orange-300"
+      className="flex items-center justify-between space-x-3 w-full text-sm p-1 text-gray-700 hover:bg-stone-200/50 font-medium rounded-lg hover:text-orange-400 focus:ring focus:ring-orange-300"
```
Comment (talboren) [inline keep-ui/app/dark-mode-toggle.tsx:46]: The toggle itself is now "different" from the rest of the items, which looks kinda weird -- maybe "adjust" it to match the new style?

File: keep-ui/components/navbar/AlertsLinks.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        {session && <CustomPresetAlertLinks session={session} />}
+        <li>
+          <LinkWithIcon href="/alerts/groups" icon={IoNotificationsOffOutline}>
+            Correlation
+          </LinkWithIcon>
+        </li>
+        <li>
+          <LinkWithIcon
+            href="/alerts/dismissed"
+            icon={IoNotificationsOffOutline}
```
Comment (talboren) [inline keep-ui/components/navbar/AlertsLinks.tsx:57]: We need different icons here

**Review by talboren — CHANGES_REQUESTED** [review]
A few minor comments I left (+the general comments). other than that, LGTM!

File: keep-ui/components/navbar/AlertsLinks.tsx (hunk 2 above)
Comment (asharonbaltazar) [reply keep-ui/components/navbar/AlertsLinks.tsx]: Resolved with https://github.com/keephq/keep/pull/1011/commits/0f73c67dd2e86959edf81c82709707237a9c70a4

File: keep-ui/app/dark-mode-toggle.tsx (hunk 1 above)
Comment (asharonbaltazar) [reply keep-ui/app/dark-mode-toggle.tsx]: Resolved with https://github.com/keephq/keep/pull/1011/commits/9ca2fce012e87371f08c3ee64ad692c5a3a2e494

**Review by talboren — APPROVED** [review]
LGTM

**Pull request merged**